src/Makefile

```diff
@@ -9,8 +9,7 @@ BAD_LINT_MSG := "Missing golangci-lint version $(WANTED_LINT_VERSION). Visit $(I
 .PHONY: check-lint check-lint-version lint load-test
 
 build:
-	go build -o corso -ldflags \
-	"-X 'github.com/alcionai/corso/src/cli/version.Version=$(shell git describe --exact-match --tags $(git rev-parse HEAD) 2>/dev/null || echo unreleased)-$(shell git rev-parse --short HEAD)'"
+	go build -o corso
```

Comment:
When building locally, there is no point in pinning a version plus more often than not, the git state will be dirty.

Reply:
The change is fine, but let's keep separate concerns split into multiple PRs in the future, please.